MigrateServicePreferenceFromLegacy/handler.ts

```diff
@@ -83,14 +84,24 @@ export const blockedsToServicesPreferences = (
     .getOrElse([]);
 
 export const MigrateServicePreferenceFromLegacy = (
-  servicePreferenceModel: ServicesPreferencesModel
+  servicePreferenceModel: ServicesPreferencesModel,
+  tracker: ReturnType<typeof createTracker>
 ) => async (context: Context, input: unknown) =>
   te
     .fromEither(
       MigrateServicesPreferencesQueueMessage.decode(input).mapLeft(
         errorsToError
       )
     )
+    // trace event
+    .map(_ => {
+      tracker.profile.traceMigratingServicePreferences(
+        _.oldProfile,
+        _.newProfile,
+        "DOING"
+      );
+      return _;
+    })
```

Comment:
Ithink we should add another event for `DONE` before .run()

Reply:
done